portfolio/src/main/java/com/google/sps/servlets/PokemonServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Handles requests sent to the /hello URL. Try runninsg a server and navigating to /hello! */
```

Comment:
I know this servlet is still not in use but still, please rename the URL and update comment